tests/integration-tests/conftest.py

```diff
@@ -908,3 +909,31 @@ def _pcluster_ami_without_standard_naming(version):
     if ami_id:
         client = boto3.client("ec2", region_name=region)
         client.deregister_image(ImageId=ami_id)
+
+
+@pytest.fixture()
+def build_image():
+    """Create a build image process."""
+    image_id_post_test = None
+    region_post_test = None
+
+    def _build_image(image_id, region, cluster_config):
+        nonlocal image_id_post_test
+        nonlocal region_post_test
+        image_id_post_test = image_id
+        region_post_test = region
+
+        pcluster_build_image_result = run_command(
+            ["pcluster", "build-image", "--id", image_id, "-r", region, "-c", cluster_config.as_posix()]
+        )
+        return pcluster_build_image_result.stdout
+
+    yield _build_image
+    if image_id_post_test:
+        pcluster_describe_image_result = run_command(
+            ["pcluster", "describe-image", "--id", image_id_post_test, "-r", region_post_test]
+        )
+        logging.info("Build image post process. Describe image result: %s" % pcluster_describe_image_result.stdout)
+
+        if "build_failed" or "error" in pcluster_describe_image_result.stdout.lower():
```

Comment:
can we add a reminder or a TODO that command output parsing needs to be revisited once the command will output a proper JSON?